backend/crud/notification.py

```
@@ -2,6 +1,8 @@
+from sqlalchemy.orm import Session, joinedload
```

Ruff (0.8.2)

`sqlalchemy.orm.joinedload` imported but unused

Remove unused import: `sqlalchemy.orm.joinedload`

(F401)